app/controllers/v1/padlock/session_controller.rb

```diff
@@ -0,0 +1,19 @@
+module V1
+  module Padlock
+    class SessionController < V1::Padlock::PadlockController
+      def create
+        @user = V1::Padlock::Sessions::Create.call(params)
+
+        return render json: nil, status: :unauthorized unless @user
+
+        render '/v1/users/show', status: :ok
+      end
+
+      def destroy
+        return render json: nil, status: :unprocessable_entity unless token
+
+        return render json: nil, head: :ok if ::Padlock::TokenDestroyer.call(token: token)
```

Comment:
does it work? :D

Reply:
Why wouldn't it?

Reply:
Because `render` checks `status` key. Change :ok, to :unprocessable_entity. Theoretically you'll see the same result.